docs/read-the-docs/source/developer_guide.rst

```diff
@@ -0,0 +1,324 @@
+Developer guide
+================
+
+Receptor is an open source project that lives at https://github.com/ansible/receptor
+
+Testing
+^^^^^^^
+
+Pull requests must pass a suite of integration tests before being merged into devel.
+
+``make test`` will run the full test suite locally. Some of the tests require access to a Kubernetes cluster, and expects to load in the kubeconfig located at ``$HOME/.kube/config``. One simple way to make these tests work is to start minikube locally before running ``make test``. See https://minikube.sigs.k8s.io/docs/start/ for more information about minikube.
+
+To skip tests that depend on Kubernetes, set environment variable ``export SKIP_KUBE=1``.
+
+Additionally, all code must pass a suite of go linters. There is a pre-commit yaml file in the receptor repository that points to the linter suite. It is best practice to install the pre-commit yaml so that the linters run locally on each commit.
```

Comment:
Capitalize "Go"

Reply:
fixed, and I changed all golang to Golang where applicable